jenkins/igt/Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+pipeline {
+  options { timestamps() }
+  agent { dockerfile { dir 'jenkins/igt/' } }
+
+  stages {
+    stage("Setup") {
+      steps {
+        script {
+          env.PIPELINE_VERSION = VersionNumber(
+             versionNumberString: '${BUILD_DATE_FORMATTED,"yyyyMMdd"}.${BUILDS_TODAY_Z}')
+        }
+        sh 'mkdir ${PIPELINE_VERSION}'
```

Comment:
I think this should not be required as the `dir()` function should be taking care of creating the directory when you call it in the next stage.